### ISSUE TITLE
Transfer Zuora cypress test from Quartz to UI

### ISSUE THREAD

**Issue by Asalem1** [influxdata/ui#1046]
Looks like there's a Cypress test in Quartz under the `editPayments` and `fillPaymentSection` tests that we can integrate into our e2e tests.

**Comment by subirjolly** [comment]
Worked on this. There are a couple of hiccups.
1. In package.json, `source .env && cypress run .....` doesn't give you access to the environment variables supplied in the .env file. In order to get that, you have to use `export`. Example: 
```
"cy:dev": "for i in $(cat ../monitor-ci/.env);do export $i;done && CYPRESS_baseUrl=https://kubernetes.docker.internal:$PORT_HTTPS cypress open --config testFiles='{cloud,shared}/**/*.*'",
```
This issue was resolved with the above mentioned line.

2. The other problem was that as we had already mocked the response, we couldn't override the response without some refactoring. There were a couple of alternatives like dependency injection of the resource caller(fetch/axios) request as well as testing the component independently(stand-alone). Those weren't the options worth investing in as we will be getting the rendering of ZuoraForm for free once we have Quartz API returning those to UI.

Discussed these scenarios with @Asalem1 and we came to the conclusion that we should pick this story back up once Quartz API is available with ZuoraParams(CreditCardParams) returning capability. Thus, moving this story back to ready in blocked state.

**Comment by Asalem1** [comment]
This is currently blocked by the need to develop a local Quartz Dockerfile to run Quartz so that the proxy can make requests to the API

**Comment by Asalem1** [comment]
This might be more tricky than i originally thought since the domain that renders the zuora form needs to be whitelisted. Gonna see if I can pair with someone on the ecommerce team to get it to render

**Comment by Asalem1** [comment]
So after meeting with members of the e-commerce team and running through the code, it looks like the existing tests were making requests to the API with specific environment variables that were then making a post request to zuora for real information to be rendered onto the page. The data returned from zuora can't be mocked out, so the new plan of action will be to add eventing around the zuora form and measure failure rates of the zuora form request that will be monitored and trigger alerts for whenever those requests fail
